sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -2,13 +2,11 @@
 
 ## 2.41.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
```

Comment:
Add the breaking changes

Reply:
The breaking changes and revapi check ignore item have been added.